README.md

```diff
@@ -58,18 +58,16 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [hello_rails_back_end] <a name="about-project"></a>
 
```

Comment:
- Optional
- [ ] Please remember to remove Microverse logo before merging this pull request. If you have your own logo you can use it there.